### PR TITLE
Add "Drownable Air" node (`maptools:drowning`)

### LINKED ISSUE #17: add node with drowning=1
like `maptools:fake_walkable` but additionally with drowning enabled

## Use-cases

* Light-weight space-simulation effects
* Suffocating gas areas

I'd like to make a PR but need a :-1: or :+1: first

### PR DESCRIPTION
**Pull request by BuckarooBanzay**
fixes #17

### CODE REVIEW

File: NODES_ITEMS.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ as they cannot be removed by hand (they can only be removed with
 | `maptools:no_interact`  | Prevents interacting through the block (opening chests, furnaces, attacking entities, …), but can still be walked through.                                             |
 | `maptools:damage_{1…5}` | :warning: Damaging blocks which damage players by 1 to 5 HP per second.                                                                                                |
 | `maptools:kill`         | :warning: Instant kill blocks (damages players by 20 HP per second).                                                                                                   |
+| `maptools:drowning`     | :warning: Simulates drowning in water                                          |
```
Comment (Calinou) [inline NODES_ITEMS.md]:
```suggestion
| `maptools:drowning`     | :warning: Simulates drowning in water.                                         |
```

**Review by Calinou — CHANGES_REQUESTED** [review]
Should be good to merge after applying the suggestion :slightly_smiling_face:

**Review by Calinou — APPROVED** [review]
Thanks!

**Comment by BuckarooBanzay** [comment]
thanks for the quick response :+1:

**Pull request merged**